src/sentry/sentry_metrics/indexer/cache.py

```diff
@@ -83,5 +85,4 @@ def delete_many(self, keys: Sequence[str], cache_namespace: str) -> None:
         self.cache.delete_many(cache_keys, version=self.version)
 
 
-# todo: dont hard code 1 as the version
 indexer_cache = StringIndexerCache(**settings.SENTRY_STRING_INDEXER_CACHE_OPTIONS)
```

Comment:
If we are going to have multiple cache instances with different settings should we move this line out of here?

Reply:
this is used in tests right now that are independent of the type of storage